scripts/install-windows-prereqs.ps1

```diff
@@ -359,6 +359,10 @@ function Install-Python3 {
     New-Directory -Path $installDir -RemoveExisting
     Move-Item -Path "$tempInstallDir\*" -Destination $installDir
     Add-ToSystemPath -Path $installDir
+
+    Start-ExecuteWithRetry -ScriptBlock {
+        pip install cmake_format
```

Comment:
Does this script already install python?

Reply:
Yes, that's what line 354 does